src/main/java/marquez/api/mappers/Mapper.java

```diff
@@ -232,6 +235,16 @@ public static RunResponse toRunResponse(@NonNull final Run run) {
         run.getArgs());
   }
 
+  public static TagResponse toTagResponse(@NonNull Tag tag) {
```

Comment:
We'll want to define the following mapper methods for a `Tag` object:

* `Mapper.toTagResponse(Tag): TagResponse` - Constructs a `TagResponse` instance from the given `Tag`
* `Mapper.toTagResponse(List<Tag>): List<TagResponse>` - Returns a list of  `TagResponse` objects from the given list of `Tag` objects
* `Mapper.toTagsResponse(List<Tag>): TagsResponse` - Constructs a `TagsResponse` instance from the given list of `Tag` objects

For an example, see [`Mapper.toNamespaceResponse`](https://github.com/MarquezProject/marquez/blob/master/src/main/java/marquez/api/mappers/Mapper.java#L74)